### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by sirielua**

### CODE REVIEW

**Comment by sirielua** [comment]
All good

File: Exercises/2-by-value.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 'use strict';
 
-const inc = null;
+const inc = num => (num + 1);
```
Comment (tshemsedinov) [inline Exercises/2-by-value.js]:
```suggestion
const inc = num => num + 1;
```

File: Exercises/4-count-types.js [hunk 2]
```diff
@@ -1,5 +1,12 @@
 'use strict';
 
-const countTypesInArray = null;
+const countTypesInArray = values => {
+  const count = {};
+  for (const val of values) {
+    const type = typeof val;
+    type in count ? count[type]++ : count[type] = 1;
```
Comment (tshemsedinov) [inline Exercises/4-count-types.js]: Improve readability, I'd prefer not to use `?` in this case. See solutions

File: package.json [hunk 3]
```diff
@@ -5,8 +5,8 @@
   "author": "Timur Shemsedinov <timur.shemsedinov@gmail.com>",
   "license": "MIT",
   "scripts": {
-    "test": "eslint ./Exercises; hpw",
-    "ci": "eslint ./Exercises && hpw"
+    "test": "eslint .\\Exercises && hpw",
+    "ci": "eslint .\\Exercises && hpw"
```
Comment (tshemsedinov) [inline package.json]: Don't change or don't commit if changed

Comment (sirielua) [reply package.json]: Thank you very much for your time. This pull request was created by mistake. My intention was to test stuff within my fork and not to bother anyone. My Apologies

Comment (tshemsedinov) [reply package.json]: That's fine)

**Pull request closed without merging**